src/test/java/model/ladder/LadderTest.java

```diff
@@ -0,0 +1,22 @@
+package model.ladder;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LadderTest {
+
+    @Test
+    public void 사다리_생성_시_주어진_사다리_정보와_일치해야_한다(){
+        int playerCount = 4;
+        int ladderHeight = 5;
+
+        Ladder ladder = new Ladder(playerCount,ladderHeight);
+
+        assertThat(ladder.getLadder()).hasSize(ladderHeight);
+        for (LadderLine ladderLine : ladder.getLadder()) {
+            assertThat(ladderLine.getLadderLine()).hasSize(playerCount);
+        }
```

Comment:
allSatisfy로 반복 검증을 수행할 수 있어요
```suggestion
assertThat(ladder.getLadder()).allSatisfy(
    ladderLine -> assertThat(ladderLine.getLadderLine()).hasSize(playerCount)
);
```

Reply:
하나 배워가네요 감사합니다! 적용해보도록 할게요 :)